src/pages/api/identities/[...id].ts

```diff
@@ -56,7 +58,9 @@ const get: Handler = async (request, response) => {
       return;
     }
 
-    const { id: identityId } = request.query;
+    const {
+      id: [identityId],
+    } = request.query;
```

Comment:
With the new naming of the file, the `id` url params is now a `string[]`, order the same way as the url
[1/2]